src/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wbpack Exercise</title>
+</head>
+<body>
+    <h1>Hello webpack!</h1>
```

Comment:
- [ ] Kindly make your HTML code look like  the [wireframe](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/images/leaderboard_wireframe.png), 
- Please note for this milestone styling and api calls is not required.

Figma Design          |  Your design     
:-------------------------:|:-------------------------:
![Screenshot_1](https://user-images.githubusercontent.com/69638013/161954308-c790e97d-fe38-42e1-96a6-d39770efdbbd.png)  |  ![Screenshot_4](https://user-images.githubusercontent.com/69638013/161954356-62ecc04d-c2d6-413a-b6d4-07e99c773f10.png)